apps/demo/src/styles.css

```diff
@@ -41,3 +41,8 @@ body {
 #root {
   height: 100vh;
 }
+
+a:focus-visible,
+[tabindex='0']:focus-visible {
+  outline: 1px dotted ButtonText;
+}
```

Comment:
Matches the focus outline set by `normalize.css` on buttons but for links and any other tabbable element (e.g. slider thumbs)